### PR TITLE
feat: renew application overview step (HL-945)

### PR DESCRIPTION
**Pull request by sirtawast**
## Description :sparkles:

* Rewrite overview of application form to conform term changes
* Extract ibantools to shared
* Fix step 1 issue with employment start date picker not setting the correct end date
* Still missing en/sv translations

### CODE REVIEW

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2334.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Review by mjturt — APPROVED** [review]

File: frontend/benefit/applicant/public/locales/en/common.json [hunk 1]
```diff
@@ -83,6 +83,7 @@
         "deMinimisAidsHeading": "Below, please fill in all the de minimis aid granted to the organisation for the current and two previous tax years.",
         "deMinimisAidsAdd": "Save",
         "deMinimisAidsRemove": "Delete",
+        "deMinimisAidsYes": "TODO",
```
[the remark this answers is not in the document]
Comment (sirtawast) [reply frontend/benefit/applicant/public/locales/en/common.json:86]: @mjturt No translations yet.

[the remark this answers is not in the document]
Comment (sirtawast) [reply frontend/benefit/applicant/public/locales/en/common.json:86]: @JanneJuhola works on those

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2334.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2334.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2334.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2334.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2334.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**